H/HDF5/build_tarballs.jl

```diff
@@ -421,12 +321,8 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    # For OpenMP we use libomp from `LLVMOpenMP_jll` where we use LLVM as compiler (BSD 
-    # systems), and libgomp from `CompilerSupportLibraries_jll` everywhere else. 
-    Dependency(PackageSpec(name="CompilerSupportLibraries_jll", uuid="e66e0078-7015-5450-92f7-15fbd957f2ae");
-               platforms=filter(!Sys.isbsd, platforms)),
-    Dependency(PackageSpec(name="LLVMOpenMP_jll", uuid="1d63c593-3942-5779-bab2-d838dc0a180e");
-               platforms=filter(Sys.isbsd, platforms)),
+    # To ensure that the correct version of libgfortran is found at runtime
+    Dependency(PackageSpec(name="CompilerSupportLibraries_jll", uuid="e66e0078-7015-5450-92f7-15fbd957f2ae")),
```

Comment:
How does CompilerSupportLibraries_jll help find the correct version of libgfortran?

Reply:
Because CompilerSupportLibraries_jll does provide libgfortran